src/pca_report_library/customer/generate_report.py

```diff
@@ -34,14 +34,12 @@
 from typing import Dict
 
 # Third-Party Libraries
-from docopt import docopt
-import pystache
-from pytz import timezone
-
-# cisagov Libraries
 from _version import __version__
 from customer.closing import closing_builder
 from customer.graphs import graph_builder
+from docopt import docopt
+import pystache
+from pytz import timezone
 from utility.time import format_timedelta_to_HHMMSS, time_to_string
```

Comment:
You have combined imports from local libraries with third-party imports- this should be corrected, along with the relative local import paths.
```suggestion
from docopt import docopt
import pystache
from pytz import timezone

from .._version import __version__
from ..utility.time import format_timedelta_to_HHMMSS, time_to_string
from .closing import closing_builder
from .graphs import graph_builder
```